### PR TITLE
Sanitize IBAN match before validation

### PR DESCRIPTION
**Pull request by acatlla**
The [IBAN pattern](https://github.com/microsoft/presidio/blob/master/presidio-analyzer/presidio_analyzer/predefined_recognizers/iban_recognizer.py#L32) detect possible IBAN codes with ` ` and `-` separators.

Sanitize theses characters from the matched text before validation to avoid #363

### CODE REVIEW

**Comment by omri374** [comment]
Hi @acatlla. Some tests are failing. Could you please check and update?

**Comment by acatlla** [comment]
> Hi @acatlla. Some tests are failing. Could you please check and update?

Sure

**Review by omri374 — APPROVED** [review]

File: presidio-analyzer/presidio_analyzer/predefined_recognizers/iban_recognizer.py [hunk 1]
```diff
@@ -63,8 +65,8 @@ def __init__(
         )
 
     def validate_result(self, pattern_text):
-        try:
-            pattern_text = pattern_text.replace(" ", "")
+        try:
```
Comment (eladiw) [inline presidio-analyzer/presidio_analyzer/predefined_recognizers/iban_recognizer.py:68]:
```suggestion
        try:
```

**Review by eladiw — APPROVED** [review]
LGTM

**Pull request merged**